auto_nag/people.py

```diff
@@ -311,6 +311,11 @@ def get_management_chain_mails(self, person: str, superior: str) -> Set[str]:
         """
         result = set()
 
+        assert person in self.people
```

Comment:
Maybe you should just use moz emails for both person and superior.
For example:
```python
person = self.get_moz_mail(person)
superior = self.get_moz_mail(superior)
assert(person and superior)
...
```

Reply:
Currently, we already passing the moz emails for both person and superior. I added the assertions to catch future errors. In this way, if we forget to use the moz email for nagging, it could be caught in the dry-run.